.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
```

Comment:
I believe our JS files use `indent_size = 2`. Using `4` as the default makes sense, but we might want to add a `*.js` override. Or perhaps we should consider changing to use `4` everywhere. We really need to "contributing guide" which defined these things. Then this file could just match whatever is defined there.

Reply:
I'm fine with either 2 or 4 really...  2 seems slightly more common.
`mkdocs\mkdocs\themes\mkdocs\js\base.js` uses 4 so I set it as that but we could change to 4.

Reply:
All the work in the recent search refactor uses 2 for JS files. We have a lot more JS files that use 2 than 4, so let's use 2.

Reply:
Cool. updated the PR to 2.